app/pods/components/commander-multiselect/component.ts

```diff
@@ -0,0 +1,32 @@
+import { action, computed } from '@ember-decorators/object';
+import Component from '@ember/component';
+import Commander from 'web-wars-ember/pods/commander/model';
+
+export default class CommanderMultiselect extends Component {
+  public model!: Commander[];
+  public value!: string[];
+  public onChange!: (value: string[]) => void;
+  public disabled ? = false;
```

Comment:
Since this has a default, it's never going to maybe be null.
```suggestion
  public disabled = false;
```

Reply:
🖖